RecoLocalCalo/HGCalRecProducers/test/testHGCalRecoLocal_cfg.py

```diff
@@ -1,7 +1,7 @@
 import FWCore.ParameterSet.Config as cms
 from Configuration.StandardSequences.Eras import eras
 
-process = cms.Process("testHGCalRecoLocal",eras.Phase2C2_timing)
+process = cms.Process("testHGCalRecoLocal",eras.Phase2C2)
```

Comment:
Lindsey should comment - but my understanding the eras for 2023D3 should not have timing